chemie/merch/models.py

```diff
@@ -0,0 +1,26 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+
+class MerchCategory(models.Model):
```

Comment:
There is no way to delete or edit merch categories. Only through admin panel. Is this intentional?